src/utils/utils.cpp

```diff
@@ -242,13 +244,12 @@ bool mp::utils::valid_mac_address(const std::string& mac)
 void mp::utils::wait_until_ssh_up(VirtualMachine* virtual_machine, std::chrono::milliseconds timeout,
                                   std::function<void()> const& ensure_vm_is_running)
 {
-    mpl::log(mpl::Level::debug, virtual_machine->vm_name,
-             fmt::format("Trying SSH on {}:{}", virtual_machine->ssh_hostname(), virtual_machine->ssh_port()));
+    mpl::log(mpl::Level::debug, virtual_machine->vm_name, "Waiting for SSH to be up");
     auto action = [virtual_machine, &ensure_vm_is_running] {
         ensure_vm_is_running();
         try
         {
-            mp::SSHSession session{virtual_machine->ssh_hostname(), virtual_machine->ssh_port()};
+            mp::SSHSession session{virtual_machine->ssh_hostname(1ms), virtual_machine->ssh_port()};
```

Comment:
Should we add a debug line after this saying something like "Found instance hostname: … port: …"?

Reply:
I don't think it's necessary since this is just testing for an ssh connection, not getting the IP address.  If we want to log the IP address discovery, that should probably go in `ip_address_for()`, but that seems to be unrelated to this PR.